src/components/molecules/Dropdown/Dropdown.js

```diff
@@ -68,11 +71,20 @@ export default class Dropdown extends HTMLElement {
         this.parentElement
           .querySelector('cod-dropdown-menu')
           .setAttribute('data-show', 'false');
+        this.parentElement
+          .querySelector('cod-dropdown-menu') 
+          .setAttribute('aria-hidden', 'true');           
```

Comment:
Just a minor comment: we can store the result of the query selector instead of performing the query twice. Like this:

```js
const dropdownMenu = this.parentElement
  .querySelector('cod-dropdown-menu');

dropdownMenu
  .setAttribute('data-show', 'false')
  .setAttribute('aria-hidden', 'true'); 
```